packages/world/src/Constants.sol

```diff
@@ -38,6 +38,11 @@ uint128 constant CRAFT_ENERGY_COST = 8100000000000000;
 uint128 constant MOVE_ENERGY_COST = 25543750000000;
 uint128 constant PLAYER_FALL_ENERGY_COST = MAX_PLAYER_ENERGY / 25; // This makes it so, with full energy, you die from a 25 block fall
 
+// Resource caps
+uint256 constant MAX_WHEAT_SEED = 5_000_000;
+uint256 constant MAX_OAK_SEED = 1_000_000;
```

Comment:
Are these from the new balancing sheet or just set arbitrarily for now?

Reply:
arbitrary for now

Reply:
created an issue to track: https://github.com/dustproject/dust/issues/181